firstvoices/backend/views/stats_views.py

```diff
@@ -0,0 +1,157 @@
+from datetime import timedelta
+
+from django.utils import timezone
+from drf_spectacular.utils import OpenApiResponse, extend_schema, extend_schema_view
+from rest_framework import viewsets
+from rest_framework.response import Response
+
+from backend.models import DictionaryEntry, Song, Story
+from backend.models.constants import Visibility
+from backend.models.dictionary import TypeOfDictionaryEntry
+from backend.models.media import Audio, Image, Video
+from backend.views import doc_strings
+from backend.views.api_doc_variables import site_slug_parameter
+from backend.views.base_views import FVPermissionViewSetMixin, SiteContentViewSetMixin
+
+
+@extend_schema_view(
+    list=extend_schema(
+        description="A list of statistics about a given site.",
+        responses={
+            200: Response,
```

Comment:
Since the API does not have a serializer I'm not quite sure what to put here, I have just added a generic response but it does not represent the actual response in the docs

Reply:
drf-spectacular provides a handy inline-serializer for instances like this where we don't have a serializer. You can depict the structure of the response in the inline serializer. Ref: https://drf-spectacular.readthedocs.io/en/latest/drf_spectacular.html#drf_spectacular.utils.inline_serializer